guides/micronaut-oauth2-cognito/metadata.json

```
@@ -1,17 +1,19 @@
 {
   "title": "Secure a Micronaut application with Cognito",
   "intro": "Learn how to create a Micronaut application and secure it with an Authorization Server provided by Cognito.",
-  "authors": ["Sergio del Amo", "Álvaro Sánchez-Mariscal"],
+  "authors": ["Sergio del Amo", "\u00c1lvaro S\u00e1nchez-Mariscal"],
```

```suggestion
  "authors": ["Sergio del Amo", ""Álvaro Sánchez-Mariscal"],
```
Why did you change this?